monitoring/README.md

```diff
@@ -485,6 +554,53 @@ git status
 git checkout master
 ```
 
+## Alerting
+
+Alerting is composed of:
+
+* Alert Rules
+* Contact Points
+* Message Templates
+* Notification Policies
+* Alert Groups (not implemented for now)
+* Mute Timings (not implemented for now)
+
+### Alert Rules
+
+Our [Alert Rules](https://grafana.com/docs/grafana/latest/alerting) are currently being serviced by Grafana.
+
+We can choose to use [Prometheus' AlertManager](https://prometheus.io/docs/alerting/latest/notification_examples/) in the future to support rich Slack messages that includes support for href links and Slack @mentions.
+
+Any alert created outside of the [auto-generated Alerts workflow](#releasing-alerts-to-production) will not have any automation applied to it. The Alerts API does not expose any way to grab the manually created Alerts' `uid` and thus, no API operations can be applied without manual intervention.
+
+All auto-generated alerts should not be deleted and should instead always be updated. All alert history is lost when an alert is deleted, however, an alert can be updated multiple times via the API while maintaining alert history. Do note that auto-generated alerts cannot be updated via the UI.
```

Comment:
question about this - if the underlying panel is edited to set the alert at a different value or if the threshold was removed what's the behavior?

Reply:
Sure thing:

* We will run a POST request using the same deterministically generated UID and update the existing alert to use the new value. This was reliable in our tests with the API.
* Good catch on the removed threshold bit! There was some manual (and CLUNKY) steps that have now been automated as part of our cronjob: 907b151ede675aef85a73a92586279c37eb32c66.